pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseSingleStageBrokerRequestHandler.java

```diff
@@ -146,6 +146,7 @@ public abstract class BaseSingleStageBrokerRequestHandler extends BaseBrokerRequ
   // if >= 0, then overrides default limit of 10, otherwise setting is ignored
   protected final int _defaultQueryLimit;
   protected final boolean _enableMultistageMigrationMetric;
+  protected final boolean _useMSQEWhenEmptySchema;
```

Comment:
(minor) Suggest keeping the names consistent
```suggestion
  protected final boolean _useMSQEToFillEmptyResponseSchema;
```

Reply:
Understood.

eb2cbc28086501a45610f4be1d496c00d47e087c